frontend/src/pages/org/browser-profiles-detail.ts

```diff
@@ -280,6 +321,33 @@ export class BrowserProfilesDetail extends TailwindElement {
       </btrix-dialog> `;
   }
 
+  private renderCrawlWorkflows() {
+    if (this.profile?.crawlconfigs?.length) {
+      return html`<ul>
+        ${this.profile.crawlconfigs.map(
+          ({ id, name }) => html`
+            <li
+              class="border-x border-b first:rounded-t first:border-t last:rounded-b"
+            >
+              <a
+                class="block p-2 transition-colors focus-within:bg-neutral-50 hover:bg-neutral-50"
+                href=${`${this.navigate.orgBasePath}/workflows/crawl/${id}`}
+                @click=${this.navigate.link}
+              >
+                ${name ||
+                html`<span class="text-neutral-400">${msg("(no name)")}</span>`}
```

Comment:
I wonder if it might be better to handle this similarly to how we handle it elsewhere in the app, i.e. use the first seed URL (and `+ int` as appropriate) if no name is available? Otherwise could end up with a list of workflows that are indistinguishable from each other here.

Reply:
The backend API only returns name and ID in profile `crawlconfigs`. Is it an easy change to return first seed?

Reply:
Ah, right! Let me take a look.

Reply:
Hm, shouldn't be a hard fix but I also don't want to block this PR further, so let's add that in a follow-up issue: https://github.com/webrecorder/browsertrix/issues/1833